tests/printer_test.py

```diff
@@ -633,7 +633,7 @@ def test_parse_dense_mlir():
     """
 
     expected = """
-    %0 = "arith.constant"() {"value" = dense<[[0.0, 1.0, 2.0], [3.0, 4.0, 5.0]]> : tensor<2x3xf64>} : () -> tensor<2x3xf64>
+    %0 = "arith.constant"() {value = dense<[[0.0, 1.0, 2.0], [3.0, 4.0, 5.0]]> : tensor<2x3xf64>} : () -> tensor<2x3xf64>
     """
```

Comment:
It's related to the test, I wanted to add a failing test from the test suite verbatim, and check that the output was the same as the input. I can take this change out.

Reply:
Yes, could we remove that change?
I prefer printing the most generic MLIR code, to avoid corner cases. For instance, here, if we have an identifier that has spaces in it, or numbers at the beginning, this will failed to be parsed again by MLIR.